tidb-cloud/changefeed-replication.md

```diff
@@ -0,0 +1,116 @@
+---
+title: TiDB Cloud Replication
+summary: Learn how to create a replica to stream data from a primary TiDB cluster to a secondary TiDB cluster.
+---
+
+# TiDB Cloud Replication
+
+TiDB Cloud Replication is a feature that allows you to create a continuously replicated readable secondary TiDB cluster for a primary TiDB cluster in TiDB Cloud. This readable secondary cluster (also known as cross-region replication, secondary replication, or geo-replication) can be in the same region as the primary TiDB cluster, or more commonly, in a different region.
+
+With TiDB Cloud Replication, you can perform quick disaster recovery of a database in the event of a regional disaster or large-scale failure, which helps achieve business continuity. Once a secondary cluster is set up, you can manually initiate geographic failover to the secondary cluster in a different region.
+
+> **Warning:**
+>
+> Currently, the **TiDB Cloud Replication** feature is in **Public Preview** with the following limitations:
+>
+> * One primary cluster can only have one replication.
+> * You cannot use a secondary cluster as a source of **TiDB Cloud Replication** to another cluster.
+> * **TiDB Cloud Replication** contradicts **Sink to Apache Kafka**](/tidb-cloud/changefeed-sink-to-apache-kafka.md) and [**Sink to MySQL**](/tidb-cloud/changefeed-sink-to-mysql.md). When **TiDB Cloud Replication** is enabled, neither the primary nor the secondary cluster can use the **Sink to Apache Kafka** or **Sink to MySQL** changefeed and vice versa.
+> * Because TiDB Cloud uses TiCDC to establish replication, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#restrictions).
```

Comment:
Can we use internal links now that the source files are in the same repo?
```suggestion
> * Because TiDB Cloud uses TiCDC to establish replication, it has the same [restrictions as TiCDC](/ticdc/ticdc-overview.md#restrictions).
```

Reply:
No, we cannot use it, because "/ticdc/ticdc-overview.md#restrictions" will be replaced with "https://docs.pingcap.com/tidb-cloud/ticdc-overview.md#restrictions" in the doc building, which will cause 404.